zerver/lib/actions.py

```diff
@@ -3306,7 +3309,14 @@ def do_rename_stream(stream: Stream, new_name: str, log: bool=True) -> Dict[str,
             name=old_name,
         )
         send_event(stream.realm, event, can_access_stream_user_ids(stream))
-
```

Comment:
It would have been better to not remove this whitespace, since it helped for readability.